added before.md file (#2861)
Codecademy/docs: content/java/concepts/date/terms/before/before.md

---
Title: '.before()'

> Description: 'Checks if a date occurs before a specified date.'

Reviewer: ```suggestion
Description: 'Checks if one date occurs before another.'
```

Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Methods'
  - 'Booleans'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The **`Date.before()`** method checks whether a date occurs before a specified date and returns a boolean value.

Reviewer: ```suggestion
The **`Date.before()`** method checks whether a date occurs before another specified date, and returns a boolean value.
```

## Syntax

```pseudo
firstDate.before(secondDate)
```

The `Date.before()` takes only one parameter:

- `secondDate`: A `Date` object to be compared chronologically with the first date.

> It returns `true` if the `firstDate` is before the `secondDate` and returns `false` if the `firstDate` is after the `secondDate`. However, if the `thirdDate` is null, it will raise a [NullPointerException](https://www.codecademy.com/resources/docs/java/errors/nullpointerexception). 

Reviewer: ```suggestion
It returns `true` if the `firstDate` is before the `secondDate` and returns `false` if the `firstDate` is after the `secondDate`. However, if a date value is null, it will raise a [NullPointerException](https://www.codecademy.com/resources/docs/java/errors/nullpointerexception). 
```
